spec/requests/organizer_can_create_event_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe 'POST /events' do
+  let(:user) { create(:user) }
+  let!(:group) { create(:group) } 
+
+  let!(:events) do 
+    3.times { create(:event, group: group) }
+  end
+
+  describe 'POST req with valid credentials' do
+    let(:user_credentials) { user.create_new_auth_token }
+    let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(user_credentials) }
+    
+    before do
+      post "/groups/#{group.id}/events", params: { event: {title: 'Craft Academy'} }, headers: headers
+    end
+
+    it 'responds with success message' do
+      expect(response_json['message']).to eq 'You have created an event successfully'
+    end
+
```

Comment:
Rephrase to 'You have successfully created an event'

Reply:
Done